tracer/test/test-applications/integrations/Samples.TraceAnnotations/TestType.cs

```diff
@@ -4,6 +4,24 @@ namespace Samples.TraceAnnotations
 {
     internal class TestType
     {
+        static TestType() { }
```

Comment:
If you do decide to add `Finalize` as an ignored method, we should add one here too (and potentially a non-finalizer called `Finalize`?